src/main/java/org/team1540/robot2018/OI.java

```diff
@@ -61,9 +77,47 @@ private static double scale(double input, double pow) {
   public static final int RIGHT_X = 4;
   public static final int RIGHT_Y = 5;
 
-  // TODO: Remove unused button
-  // wrist to forward 45 degrees
-  // static Button copilotX = new JoystickButton(copilot, X);
+  public OI() {
```

Comment:
This can be replaced with a static initializer:
```java
static {
  // INTAKE
  OI.intakeSequenceButton.whenPressed(new IntakeSequence());
  OI.ejectButton.whenPressed(new JoystickEject());

  // and so on
}
```

Also, are we sure we want to be initializing buttons in `OI` now? I was under the impression that we should do that in `robotInit()`–nothing wrong with `OI` but maybe @ambercmerrill should check it.

Reply:
Apparently initializing buttons in OI is the convention, I can't find the actual source that says this but other teams using wpilib also initialize button commands in OI.

Reply:
Seems good. Static initializer comment still stands.

Reply:
Replaced with static initializer